templates/submission/status.html

```diff
@@ -52,6 +53,15 @@
         });
         </script>
     {% endif %}
+    <script type="text/javascript">
+        $(function () {
+            $(document).keydown(function(e) {
+                if ((e.metaKey || e.ctrlKey) && e.which == 13) {
```

Comment:
Should have a comment for those not fluent in keycode numbers :)

Reply:
Done